jenkins/release-workflows/release-chores.jenkinsfile

```diff
@@ -215,6 +216,25 @@ parameters {
                 }
             }
         }
+        stage('buildRC') {
+            when {
+                expression { params.RELEASE_CHORE == 'buildRC' }
+            }
+            steps {
+                script {
+                    buildRC(
+                        version: "${params.RELEASE_VERSION}",
+                        product: "${params.ACTION}"
+                        )
+                    // Add RC details comment if both products are built    
+                    if (params.ACTION == 'both') {
+                        addRcDetailsComment(
+                            version: "${params.RELEASE_VERSION}"
```

Comment:
Let me know if this needs to be kept manual for now. This functionality will add comment when both RCs are built and are successful.
Thanks!